### PR TITLE
Make the GUID fuzzy match in the admin speedy.

### LINKED ISSUE #2951: "By GUID" filter in Link admin is slow/intensive
find out why. is it triggering counts?

### PR DESCRIPTION
**Pull request by rebeccacremona**
Fixes https://github.com/harvard-lil/perma/issues/2951.

Before:
image

After:
image

### CODE REVIEW

**Review by bensteinberg — APPROVED** [review]
Nice!

File: perma_web/perma/settings/deployments/settings_common.py [hunk 1]
```diff
@@ -137,6 +137,7 @@
     # built in apps
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.postgres',
```
Comment (lizadaly) [inline perma_web/perma/settings/deployments/settings_common.py:140]: Is this an app in the Django sense?

Comment (rebeccacremona) [reply perma_web/perma/settings/deployments/settings_common.py:140]: Weird, right? It's required, for a person to use `OpClass` https://docs.djangoproject.com/en/4.1/ref/contrib/postgres/indexes/#opclass-expressions

Comment (lizadaly) [reply perma_web/perma/settings/deployments/settings_common.py:140]: TIL! 👍

**Pull request merged**